ui/apps/platform/src/Containers/VulnMgmt/List/NodeComponents/VulnMgmtListNodeComponents.js

```diff
@@ -50,7 +50,7 @@ export function getComponentTableColumns(workflowState) {
             sortField: componentSortFields.COMPONENT,
         },
         {
-            Header: `CVEs`,
+            Header: `Node CVEs`,
```

Comment:
I think we need to feature flag the name, the entityType line, and the link building logic below ,for this.